jest.config.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
   testMatch: ["**/__tests__/**/*.spec.ts"],
-  testPathIgnorePatterns: ["/assembly/", "/node_modules/"],
```

Comment:
Don't do this.  We need jest to ignore this folder.

Reply:
got it!